### PR TITLE
EASYOPAC-336 - Fix extended search for rosbib.

### PR DESCRIPTION
**Pull request by makcumm**

### CODE REVIEW

**Comment by guddo** [comment]
You are changing the core module ting_search to accomplish this?!

File: modules/ting_search/ting_search.module [hunk 1]
```diff
@@ -925,15 +925,23 @@ function ting_search_submit($form, &$form_state) {
...
+  drupal_alter('ting_search_fields', $extended_fields);
+
+  foreach ($extended_fields as $name => $data) {
+    if (isset($data['value'])) {
+      $fields[] = $data['value'];
+    }
+    else {
+      $value = drupal_array_get_nested_value($form_state['values'], $data['parents']);
+      if (!empty($value)) {
+        $fields[] = $data['index'] . '=' . (strpos($value, ' ') ? '"' . $value . '"' : $value);
```
Comment (AnatolyMuntean) [inline modules/ting_search/ting_search.module]: This can lead to unexpected results, since strpos() can return 0 as well (beginning of the string).

**Comment by AnatolyMuntean** [comment]
+1 here. Suggest to dig whether this is really ting_search issue. AFAIK extended search is a separate module.

**Comment by makcumm** [comment]
Form task comments: "Problems may be because of ting search changes, that were not merged into rosbib branch" (https://inlead.atlassian.net/browse/EASYOPAC-336?focusedCommentId=46916&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-46916)

For now on ting_extended_search not compatible with ting_search from easySuite:ding2-master, so to make ting_extended_seaerch for easyting was made changes to ting_searech: https://github.com/easySuite/ding2/blob/easyopac/modules/ting_search/ting_search.module#L932

That's why i made changes in ting_search module, otherwise i think we will have to override ting_extended_search, to be compatible with ding2:master. Don't forget that this PR made to rosbib branch, not master.

**Comment by guddo** [comment]
Ok, so it turns our ting_extended_search is not working on a plain ding2 without modifications. Changes was made in another PR.

@makcumm: Then I suggest that we merge https://github.com/ding2/ding2/pull/299 into rosbib branch instead of making a new PR with almost similar code.

**Pull request closed without merging**